synapse/config/ratelimiting.py

```diff
@@ -102,6 +102,16 @@ def read_config(self, config, **kwargs):
             defaults={"per_second": 0.01, "burst_count": 3},
         )
 
+        # Ratelimit cross-user key requests:
+        # * For locacl requests this is done by the sending devie.
+        # * For requests received over federation this is done by the origin.
```

Comment:
```suggestion
        # * For local requests this is keyed by the sending device.
        # * For requests received over federation this is keyed by the origin.
```

It took me a bit to understand that you meant "keyed by", rather than "the ratelimiting is done by"

Reply:
Ah, that was quite a few typos in there. 😢 Thank you!

Reply:
It's rare that I'm the one to spot typos :D